bin/clang.nix

```diff
@@ -4,9 +4,12 @@ let
 in rec {
   clangEnv = stdenv.mkDerivation rec {
     name = "clang-env";
+    CLANG_PATH = pkgs.clang + "/bin/clang";
+    CLANGPP_PATH = pkgs.clang + "/bin/clang++";
     buildInputs = [
       stdenv
       pkgs.boehmgc
```

Comment:
Is it really `clang.nix` if the thing includes `boehmgc`. Shouldn't it something along the lines of `scala-native.nix` instead?